File: Jetlink.Example.Objective-C/JetlinkSample/ViewController.m
//
//  ViewController.m
//  JetlinkSample
//
//  Created by Kaan Tezgel on 24/02/17.
//  Copyright © 2017 VesLabs. All rights reserved.
//

#import "ViewController.h"
@import JetlinkSDK;

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
}
- (IBAction)ChatClick:(id)sender {
    [JetlinkApp activateChatPanel];
}
- (IBAction)FAQClick:(id)sender {
    [JetlinkApp activateFAQPanel];
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


@end

File: Jetlink.Example.Swift3/Jetlinksample/ViewController.swift
//
//  ViewController.swift
//  Jetlinksample
//
//  Created by Kaan Tezgel on 28/02/17.
//  Copyright © 2017 VesLabs. All rights reserved.
//

import UIKit
import JetlinkSDK

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
    
    @IBAction func ChatClick(_ sender: UIButton) {
        JetlinkApp.activateChatPanel()
    }
    
    @IBAction func FAQClick(_ sender: UIButton) {
        JetlinkApp.activateFAQPanel()
    }
    
    
}

